plugin/kubernetes/object/endpoint.go

```diff
@@ -44,10 +44,10 @@ type EndpointPort struct {
 func EndpointsKey(name, namespace string) string { return name + "." + namespace }
 
 // ToEndpoints converts an api.Endpoints to a *Endpoints.
-func ToEndpoints(obj interface{}) (*api.Endpoints, *Endpoints) {
+func ToEndpoints(obj interface{}) (*api.Endpoints, interface{}) {
```

Comment:
This is kind of a weird change. I think it would be more readable if we changed the signature to `func ToEndpoints(*api.Endpoints) (*Endpoints)`, do the casting in the caller, and just not call it if `obj` is not a `*api.Endpoints`.

Reply:
Assuming as I believe `ToEndpoints` is only called once anyway.

Reply:
yeah - i was just making this change. :)